packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -336,6 +340,13 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 					if (method === "network") {
 						props.cacheEntryAge = undefined;
 					}
+					if (
+						this.firstVersionCall &&
+						method === "network" &&
+						!this.hostPolicy.summarizerClient
```

Comment:
I wonder why summarizer should be special cased here. There might be different logic (and maybe it will change over time) if we load from cache at all, or not. But if we actually loaded from network, it does not feel like there would be any difference for summarizer vs. interactive client.
I might be missing something obvious :)